### PR TITLE
Requirements.txt attempt

### PR DESCRIPTION
**Pull request by Steven-Palayew**
Delete lock, toml, add requirements.txt, add test script. There may be some idea files that need to be added to gitignore.

Closes #4.

### CODE REVIEW

**Comment by JohnGiorgi** [comment]
@Steven-Palayew Great, this works for me. I would recommend adding a readme with instructions for installing, but if you want you can do that in another PR. Can you add `.idea` to the gitignore?

File: pathway_abstract_classifier/classify_article.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Import Libraries
+import ktrain
+import sklearn
```
Comment (JohnGiorgi) [inline pathway_abstract_classifier/classify_article.py:3]: `sklearn` is unused so I would drop it.

File: pathway_abstract_classifier/classify_article.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Import Libraries
```
Comment (JohnGiorgi) [inline pathway_abstract_classifier/classify_article.py:1]: Wouldn't bother commenting this, its clear that is whats happening

Comment (JohnGiorgi) [inline pathway_abstract_classifier/classify_article.py:1]:
```suggestion
```



Wouldn't bother commenting this, its clear that is whats happening

File: pathway_abstract_classifier/classify_article.py (hunk 1 above)
Comment (JohnGiorgi) [inline pathway_abstract_classifier/classify_article.py:3]:
```suggestion
```

`sklearn` is unused so I would drop it.

**Comment by Steven-Palayew** [comment]
Thanks! Tried to address these suggestions in latest commit. Let me know if that works

File: .gitignore [hunk 3]
```diff
@@ -78,6 +78,9 @@ target/
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# IDEA files
+/.idea
```
Comment (JohnGiorgi) [inline .gitignore:82]:
```suggestion
.idea/
```

the .idea is still showing up (you can see for self in the commits. I think you have the syntax backwards.

by the way, you can directly commit this suggestion to save time.

Comment (Steven-Palayew) [reply .gitignore]: Hmm yeah I was seeing both on Stack Overflow. Is it working now?

Comment (JohnGiorgi) [reply .gitignore]: I still see the files under “Files changed”. You might need to explicitly delete them and push that

**Pull request merged**